packages/ui/core-components/src/lib/unsorted/ui/Image.svelte

```diff
@@ -0,0 +1,43 @@
+<script context="module">
+	export const evidenceInclude = true;
+</script>
+
+<script>
+	import { processDimension, toBoolean } from '../../utils.js';
+	import { cn } from '$lib/utils.js';
+
+	let className = undefined;
+	export { className as class };
+
+	export let url = ''; // URL of the image
+	export let description = ''; // Description for accessibility
+	export let width = ''; // Width of the image
+	export let height = ''; // Height of the image
```

Comment:
What happens if width and height are left unset?

Reply:
The image defaults to original dimensions